### ISSUE TITLE
Requirement name is bolded but not emphasized

### ISSUE THREAD

**Issue by sfoslund** [microsoft/accessibility-insights-web#5247]
## Describe the bug

Details view requirement results text does not meet accessibility requirements. Automatic failure results are listed in the details view with the name of the requirement bolded and followed by details. The requirement name appears bolded visually but is not surrounded by an emphasis tag, which is a violation. This repros in both automated tab stops details view results as well as fast pass details view results. 

**To Reproduce**

1. Run automated tab stops or fastpass on any target page with requirement failures. 
2. See details view failure results, note that failure name is bolded but not emphasized in html. 

## Expected behavior

Bolded text should be surrounded by an emphasis tag in the html

## Screenshots

[Screenshot of bolded requirement failure, which is not highlighted with an emphasis tag in the html]
![KeyboardNavTitle](https://user-images.githubusercontent.com/16010855/157343471-c3369c9d-633b-412e-a819-179b5515671d.png)

**Comment by sfoslund** [comment]
Fixed by #5249
